modules/etcd-tuning-parameters.adoc

```diff
@@ -0,0 +1,137 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/recommended-performance-scale-practices/recommended-etcd-practices.adoc
+
+:_content-type: PROCEDURE
+[id="etcd-tuning-parameters_{context}"]
+= Setting tuning parameters for etcd
+
+You can set the control plane hardware speed to `""` (the default), `"Standard"`, or `"Slower"`.
+
+The default setting allows the system to decide which speed to use. This value enables upgrades from versions where this feature does not exist, as the system can select values from previous versions.
+
+By selecting one of the other values, you are overriding the default. If you see many leader elections due to timeouts or missed heartbeats and your system is set to `""` or `"Standard"`, set the hardware speed to `"Slower"` to make the system more tolerant to the increased latency.
+
+:FeatureName: Timer selectable profiles for etcd
+include::snippets/technology-preview.adoc[]
+
+[id="etcd-changing-hardware-speed-tolerance_{context}"]
+== Changing hardware speed tolerance
+
+To change the hardware speed tolerance for etcd, complete the following steps.
+
+.Procedure
+
+. Check to see what the current value is by entering the following command:
++
+[source, terminal]
+----
+$ oc describe etcd/cluster | grep "Control Plane Hardware Speed"
+----
++
+.Example output
+[source,terminal]
+----
+Control Plane Hardware Speed:  <VALUE>
+----
++
+[NOTE]
+====
+If the output is empty, the field has not been set and should be considered as the default ("").
+====
+
+. Change the value:
++
+[source,terminal]
+----
+oc patch etcd/cluster --type=merge -p '{"spec": {"controlPlaneHardwareSpeed": ""}}'<1>
```

Comment:
I'm not sure what our precedence is for this, but maybe `"controlPlaneHardwareSpeed": "VALUE"`  would be more clear in conjunction with the valid values below.